@here/harp-mapview-decoder/lib/TileDataSource.ts

```diff
@@ -72,11 +72,10 @@ export interface TileDataSourceOptions {
     concurrentDecoderWorkerCount?: number;
 
     /**
-     * Optional, default copyright information of tiles provided by this data source.
-     *
-     * Implementation should provide this information from the source data if possible.
+     * Optional copyright info provider for tiles provided by this data source.
+     * If omitted, default provider with static HERE copyright and current year will be used.
```

Comment:
Please note in current implementation (and this patch doesn't change this), it's not true ...

When evaluating copyright info, priority is given to information directly embedded in tiles (that comes from `DecodedTile), so both `copyrightProvider` and `copyrightInfo` are just fallbacks for data sources with no `source` in OMV/MVT properties.

So this comment (after change is misleading), so it must be either updated and or whole approach should be changed.

That's another reason, why this whole `copyrignt(Info/Provider)` should be optional.